### PR TITLE
adds slack channel link to CONTRIBUTING.md

### LINKED ISSUE #930: Add kube-state-metrics slack channel to CONTRIBUTING.md
We have a [#kube-state-metrics](https://kubernetes.slack.com/messages/CJJ529RUY) that can be used by people who have questions about the project. This offers an alternative to raising GitHub issues for the same purpose.

### PR DESCRIPTION
**Pull request by manoelstilpen**
**What this PR does / why we need it**:
It adds the slack `kube-state-metrics` link to CONTRIBUTING.md. It's helpful for those who wants to contribute and needs to clarify some doubts about this project.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #930

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -20,6 +20,10 @@ If your repo has certain guidelines for contribution, put them here ahead of the
 
 - [Mentoring Initiatives](https://git.k8s.io/community/mentoring) - We have a diverse set of mentorship programs available that are always looking for volunteers!
 
+## Contact Information
+
+- [Slack channel](https://kubernetes.slack.com/?redir=%2Fmessages%2FCJJ529RUY) - A channel that can be used by people who have questions about the project. It offers an alternative to submitting a Github issue;
+
 <!---
```
Comment (tariq1890) [inline CONTRIBUTING.md]: Can you delete this markdown comment?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -20,6 +20,10 @@ If your repo has certain guidelines for contribution, put them here ahead of the
 
 - [Mentoring Initiatives](https://git.k8s.io/community/mentoring) - We have a diverse set of mentorship programs available that are always looking for volunteers!
 
+## Contact Information
+
+- [Slack channel](https://kubernetes.slack.com/?redir=%2Fmessages%2FCJJ529RUY) - A channel that can be used by people who have questions about the project. It offers an alternative to submitting a Github issue;
```
Comment (tariq1890) [inline CONTRIBUTING.md]: Please add this text as well - 
```
[Join Slack](http://slack.k8s.io) to sign up and join the Kubernetes Slack. Please make sure to read our [Slack Guidelines](https://github.com/kubernetes/community/blob/master/communication/slack-guidelines.md) before participating.
```

Sorry, the link I provided in the GH issue is not the proper one, but a redirect.
```suggestion
-  The [kube-state-metrics slack channel](https://kubernetes.slack.com/messages/CJJ529RUY) provides an effective communication platform to reach out to members and other users of the project. It offers an alternative to submitting a GitHub issue for when you have questions and issues.
```

**Comment by tariq1890** [comment]
Thank you for your PR @manoelstilpen :).

File: CONTRIBUTING.md (hunk 1 above)
Comment (manoelstilpen) [reply CONTRIBUTING.md]: done

File: CONTRIBUTING.md (hunk 2 above)
Comment (manoelstilpen) [reply CONTRIBUTING.md]: done

**Comment by manoelstilpen** [comment]
@tariq1890 you're welcome. Your suggestions were applied

**Pull request merged**